plugins/chat/assets/javascripts/discourse/services/chat-notification-manager.js

```diff
@@ -36,6 +35,17 @@ export default class ChatNotificationManager extends Service {
     withPluginApi("0.12.1", (api) => {
       api.onPageChange(this._pageChanged);
     });
+
+    this._pageChanged();
+
+    this._chatPresenceChannel.on(
+      "change",
+      this._subscribeToCorrectNotifications
+    );
+    this._corePresenceChannel.on(
```

Comment:
Are those callbacks properly cleaned up on destroy?

Reply:
You are right, it seems there are not, I thought it was backed in presence code, but apparently not :+1:

Reply:
Pushed a fix for it